### ISSUE TITLE
[Question] Is it necessary to activate Apple Secure Boot?

### ISSUE THREAD

**Issue by 3arthur6** [perez987/OpenCore-and-UEFI-Secure-Boot#7]
I followed successfully your guide and was able to boot up to OpenCanopy picker list.
But MacOS installation isn't showed in the list, I can see an access denied error for APFS in the logs.

So I would ask if it is necessary to activate Apple Secure Boot or anything else in the config.plist?

Thanks for the guide anyway!

**Comment by perez987** [comment]
@3arthur6 
It is not clear for me why you can't boot the APFS filesystem. UEFI Secure Boot (USB) and Apple Secure Boot (ASB) are different things. In fact, macOS ignore all about USB because in the Mac world it doesn't exist, macOS has ASB (and firmware password) instead.
Theoretically you can have USB without ASB and viceversa.
Did you do any other change in the OpenCore EFI? Only signing OpenCore files and enabling USB? What is your macOS version? Which are the values of UEFI >> APFS >> MinDate and MinVersion?

**Comment by 3arthur6** [comment]
Ok thanks! I couldn't find informations about this..
Yes I only signed efi files and enabled UEFI Secure Boot with MinDate and MinVersion at -1. It's macOS 10.13
I made lot of try in config.plist but always get this access denied.

**Comment by perez987** [comment]
@3arthur6 
Tomorrow I’ll try and comment. To see if it happens the same to me.

**Comment by 3arthur6** [comment]
I finally get it to boot by enabling _DisableSecurityPolicy_ and _ProtectUefiServices_ quirks.
I probably lose all security advantage of secure boot with _DisableSecurityPolicy_ enabled but anyway this was not the intent.

**Comment by perez987** [comment]
@3arthur6 
Ok, I can reproduce your issue, OpenCore can't boot macOS with UEFI Secure Boot enabled if Apple Secure Boot is disabled. It needs medium security as a minimum. Maybe is the expected behaviour.
Indeed, enabling _DisableSecurityPolicy_ fixes the issue. I usually have _ProtectUefiServices_ enabled.

**Comment by perez987** [comment]
@3arthur6 

You've already read Mike Beaton's comment on the bugtracker, but I'm writing it here as well to leave the issue well completed.

Although it's not so obvious from reading these instructions, it appears that OpenCore does not allow access to APFS systems running macOS if UEFI Secure Boot is enabled but Apple Secure Boot is disabled. At least as of macOS 10.13, when Apple Secure Boot was released: read the chapter 12.2 UEFI Secure Boot of the OpenCore setup PDF.

It is also not possible to activate Apple Secure Boot on HFS+ systems.

In short, with Apple Secure Boot disabled, if you enable UEFI Secure Boot you cannot access APFS disks from OpenCore.

**Comment by 3arthur6** [comment]
Huh but I just explained that yes you can.
UEFI Secure Boot without Apple Secure Boot boots fine APFS by enabling _DisableSecurityPolicy_ and _ProtectUefiServices_ quirks.
Not sure for HFS or other configs.

**Comment by perez987** [comment]
@3arthur6 
Yes, you can, although it is not a recommended setting from a security point of view.

In the OpenCore configuration you can read:
`DisableSecurityPolice: disables various security features of the firmware, defeating the purpose of any kind of Secure Boot. Do NOT enable if using UEFI Secure Boot` (I understand what is meant "if using UEFI Secure Boot + Apple Secure Boot").

But, as you say, it's technically possible to have UEFI Secure Boot without Apple Secure Boot, enabling those quirks, for anyone who wants to work in this way.
